schedule/yast/iscsi_ibft.yaml

```diff
@@ -9,33 +9,23 @@ vars:
   NICTYPE: user
   YUI_REST_API: 1
 schedule:
-  - installation/bootloader_start
-  - installation/setup_libyui
-  - installation/access_beta_distribution
-  - installation/product_selection/install_SLES
-  - installation/licensing/accept_license
-  - installation/iscsi_configuration
-  - installation/registration/register_via_scc
-  - installation/module_registration/register_module_desktop
-  - installation/add_on_product/skip_install_addons
-  - installation/system_role/accept_selected_role_SLES_with_GNOME
-  - installation/partitioning
-  - installation/partitioning_iscsi
-  - installation/partitioning/accept_proposed_layout
-  - installation/clock_and_timezone/accept_timezone_configuration
-  - installation/authentication/use_same_password_for_root
-  - installation/authentication/default_user_simple_pwd
-  - installation/installation_settings/validate_default_target
-  - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/launch_installation
-  - installation/confirm_installation
-  - installation/performing_installation/perform_installation
-  - installation/logs_from_installation_system
-  - installation/performing_installation/confirm_reboot
-  - installation/grub_test
-  - installation/first_boot
-  - console/system_prepare
-  - console/hostname
-  - console/force_scheduled_tasks
-  - shutdown/grub_set_bootargs
-  - installation/validation/ibft
+  registration:
+    - installation/iscsi_configuration
+    - installation/registration/register_via_scc
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
+  suggested_partitioning:
```

Comment:
this should be under `guided_partitioning` dropping last test modules `accept_proposed_layout` to avoid repetition.